app/src/controllers/submission.ts

```diff
@@ -111,18 +111,22 @@ const controller = {
               applicationStatus: APPLICATION_STATUS_LIST.NEW,
               companyNameRegistered: data.companyNameRegistered,
               contactEmail: data.contactEmail,
+              contactPreference: camelCaseToTitleCase(data.contactPreference),
+              projectName: data.projectName,
+              projectDescription: data.projectDescription,
               contactPhoneNumber: data.contactPhoneNumber,
               contactName: `${data.contactFirstName} ${data.contactLastName}`,
+              contactApplicantRelationship: camelCaseToTitleCase(data.contactApplicantRelationship),
               financiallySupported: Object.values(financiallySupportedValues).includes(true),
               ...financiallySupportedValues,
               intakeStatus: toTitleCase(data.form.status),
               locationPIDs: data.parcelID,
               latitude: data.latitude,
               longitude: data.longitude,
               naturalDisaster: data.naturalDisasterInd,
-              projectName: data.projectName,
               queuePriority: parseInt(data.queuePriority),
               singleFamilyUnits: maxUnits,
+              isRentalUnit: camelCaseToTitleCase(deDupeUnsure(data.isRentalUnit)),
```

Comment:
'Unsure' is a pointless value. 
maybe the CHEFS data should be converted `true|false|null` in the database. The postgres `boolean` type .. and prisma and joi.. support that.. (eg  for knex table.boolean('isRentalUnit')
The isTruthy utils function might handle the deduping as well.